examples/ExampleCommands.php

```diff
@@ -8,19 +10,39 @@
 use Symfony\Component\Console\Input\InputOption;
 
 /**
- * Example RoboFile.
+ * Example Robo plugin commands.
+ *
+ * Commandfiles from any Composer project may be registered to load with Robo.
+ * To use, follow the steps below:
+ *
+ * 1. Add an entry to the 'psr-4' section of the 'autoload' section in
+ *    your composer.json file. The namespace must end with `RoboPlugin\\`
+ *
+ *     "autoload":{
+ *         "psr-4":{
+ *             "MyProject\\":"src",
+ *             "MyProject\\RoboPlugin\\":"plugin"
+ *         }
+ *     },
  *
- * To test:
+ * 2. Add command files to the specified directory. The command files must
+ *    have names ending `Commands.php`. They must either be immediately
+ *    inside the directory specified in the psr-4 autoload section, or they
+ *    may be anywhere inside a directory named `Commands`.
```

Comment:
You can put hooks in any command file. If you prefer, we could also define a default naming convention for files that are intended to have only hooks in them.

Note, however, that there is no difference in behavior between command files and hook files; you could put commands in hook files, or visa-versa.